src/blue-theme.scss

```diff
@@ -0,0 +1,19 @@
+@charset "utf-8";
+
+// Set brand colors
+$democracy-blue: #2e4272;
+
+// Update Bulma's global variables
+$primary: $democracy-blue;
+$tooltip-background-color: $democracy-blue;
+
+@import "~bulma/bulma";
+@import "~bulma-tooltip";
+
+[class*=" lavinia-logo"],
+[class^="lavinia-logo"] {
+    stroke: findColorInvert($primary);
+    fill: findColorInvert($primary);
+    width: 90px;
+    height: 28px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)